com.synology.CloudStationBackup.metainfo.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.synology.CloudStationBackup</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://www.synology.com/en-global/company/legal/terms_EULA</project_license>
+  <name>Synology Cloud Station Backup</name>
+  <developer_name>Synology</developer_name>
+  <url type="homepage">https://www.synology.com</url>
+  <summary>Real-time backup saving you from massive data loss</summary>
+  <description>
+    <p>Featuring real-time incremental backup technology, your data stored on a PC can be backed up to a Synology NAS instantly, using minimal system resources, bringing a truly comprehensive and secure private cloud experience.</p>
+    <p>NOTE: This wrapper is not verified by, affiliated with, or supported by Synology Inc.</p>
+    <p>NOTE: A system/status icon tray is required for this package to function correctly.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://www.synology.com/img/dsm/desktop_backup/pc_protection_bg.png</image>
+      <caption>Cloud Station Backup for PC protection</caption>
+    </screenshot>
```

Comment:
```suggestion
  <screenshots>
    <screenshot>
      <image type="source">https://www.synology.com/img/dsm/desktop_backup/pc_protection_bg.png</image>
      <caption>Cloud Station Backup for PC protection</caption>
    </screenshot>
```

Reply:
Misread this. Should be resolved with 50552ff.